### DISCUSSION TITLE
文件下载到一般报错无法继续下载

### DISCUSSION THREAD

**Discussion by LuckDogHu** [houtianze/bypy#668] [Q&A]
<img width="1010" alt="8dd6a29b4d5f2399ba74b1f33f4bd5b" src="https://github.com/houtianze/bypy/assets/45897157/4eb05b90-c786-4917-96d0-655bfe734f09">
<img width="1010" alt="2709fc4508bf03b169b05a50a5a1fe8" src="https://github.com/houtianze/bypy/assets/45897157/a5835399-66b3-4be2-b715-c6914695d3d9">

**Comment by houtianze** [comment]
看截图是硬盘没空间了吧？

**Reply by LuckDogHu** [reply]
确实是硬盘空间不足,感谢作者🙏
